dist/index.html

```diff
@@ -12,20 +12,22 @@
         <img class="nav-logo pointer" id="logo" src="./images/home-button.png" alt="Take Me Home" tabindex="0">
         <menu class="nav-links">
             <label for="searchBar" class="off-page">Search Recipes</label>
-            <input type="search" placeholder="Search Recipes..." class="search-bar" id="searchBar">
-            <div class="dropdown pointer" tabindex="0">
-                <li class="dropbtn filter-header" id="filterHeader">Filter Recipes</li>
-                  <div class="dropdown-content" id="filterDropdown">
-                    <li>Appetizer</li>
-                    <li>Breakfast</li>
-                    <li>Main Course</li>
-                    <li>Salad</li>
-                    <li>Side Dish</li>
-                    <li>Dip</li>
-                    <li>Sauce</li>
-                 </div>
-            </div>
-            <li id="navMyRecipes" class="pointer" tabindex="0">My Recipes</li>
+            <input type="search" placeholder="Search Recipes..." class="input-bar search-bar hover" id="searchBar">
+            <form>
```

Comment:
Love the redesign here. Enables tabability much more easily and looks clean!

Reply:
This looks much sleeker!